pkg/node/manager/node_manager.go

```diff
@@ -37,9 +40,24 @@ func NewNodeManager(params NodeManagerParams) *NodeManager {
 	return &NodeManager{
 		resourceMap: concurrency.NewStripedMap[models.Resources](resourceMapLockCount),
 		nodeInfo:    params.NodeInfo,
+		heartbeats:  params.Heartbeats,
 	}
 }
 
+func (n *NodeManager) Start(ctx context.Context) error {
+	log.Ctx(ctx).Info().Msg("Node manager started")
```

Comment:
log this after the thing starts, it technically hasn't started yet, and it my not start if `n.heartbeats` is `nil`